packages/teleport/src/SideNav/SideNavItem.tsx

```diff
@@ -41,9 +48,11 @@ const fromTheme = ({ $nested = false, theme = defaultTheme }) => {
   };
 
   if ($nested) {
+    css.fontSize = '11px';
```

Comment:
just a comment, not asking for change: i looked at the menu and 11px is too small. 14px to 11px was such a big jump my eyes had a hard time adjusting from 14px.